core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -8099,14 +8099,24 @@ private void checkLiteral2(String expression, String expected) {
         + "FROM \"foodmart\".\"product\"";
     final String expectedStarRocksSql = "SELECT APPROX_COUNT_DISTINCT(`product_id`)\n"
         + "FROM `foodmart`.`product`";
+    final String expectedMssql = "SELECT APPROX_COUNT_DISTINCT([product_id])\n"
+        + "FROM [foodmart].[product]";
+    final String expectedPhoenix = "SELECT APPROX_COUNT_DISTINCT(\"product_id\")\n"
```

Comment:
is this exactly the same as ApproxQuota above?
Is there a benefit having two identical strings?